frontend/views/GameDetailedView.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { AppStore } from '../storage/common';
+import { Game } from '../storage/models/Game';
+import { GameSeries } from '../storage/models/GameSeries';
+import { Image } from '../storage/models/Image';
+import { Narrow } from '../components/styles/Narrow';
+import { PageHeader } from '../components/styles/PageHeader';
+import { PageTitle } from '../components/styles/PageTitle';
+import { PageSubtitle } from '../components/styles/PageSubtitle';
+import { PageSection } from '../components/styles/PageSection';
+import { SectionHeader } from '../components/styles/SectionHeader';
+import { GameBox } from '../components/GameBox';
+import { RouteComponentProps } from 'react-router-dom';
+
+interface MatchParams {
+	chosenGame: string;
+}
+interface GameDetailevViewProps {
+	games: Map<number, Game>;
+	gameSeries: Map<number, GameSeries>;
+	images: Map<number, Image>;
+	chosenGameId: number;
+}
+interface GameDetailedViewState {
+	chosenGame: Game;
+}
+
+export class GameDetailedView extends React.Component<GameDetailevViewProps, GameDetailedViewState> {
+	chosenSeries: GameSeries;
+	chosenImage: Image;
+	constructor(props: GameDetailevViewProps) {
+		super(props);
+		this.state = {
+			chosenGame:
+				props.games.get(props.chosenGameId) ||
+				new Game({ title: 'Undefined', id: -1, seriesId: -1, imageId: -1, description: 'undefined', analysis: 'undefined' }),
+		};
+		this.chosenSeries =
+			props.gameSeries.get(this.state.chosenGame.seriesId) ||
+			new GameSeries({ id: -1, name: 'undefined', description: 'undefined', wikiUrl: 'undefined' });
+		this.chosenImage = props.images.get(this.state.chosenGame.imageId) || new Image({ id: -1, name: 'undefined', fileUrl: 'undefined' });
```

Comment:
Instead of creating a `new Image()` here it makes more sense to have a generic "Missing image" instance you can use anywhere. You can store it as a static member in `Image` class for now (or export a const with the default instance. Just create some temporary gray image with a question mark in the middle or something).